components/tag/TagsInput.js

```diff
@@ -1,37 +1,37 @@
-import { useRef } from "react";
 import XMarkIcon from "@heroicons/react/20/solid/XMarkIcon";
 import Input from "../form/Input";
 
-export default function TagsInput({ tags, onTagAdd, onTagRemove }) {
-  const inputRef = useRef(null);
+export default function TagsInput({ tags, onTagAdd, onTagRemove, tagInputRef }) {
```

Comment:
If this name was kept the same there would be less changes in the file. I think `inputRef is still descriptive in the scope of this function

Reply:
so should I the `tagInputref` to `inputRef`?

Reply:
Yes please 👍